src/grid/molgrid.py

```diff
@@ -18,36 +18,32 @@
 # along with this program; if not, see <http://www.gnu.org/licenses/>
 # --
 """Molecular grid class."""
+
+
 from grid.atomic_grid import AtomicGrid
 from grid.basegrid import Grid, SimpleAtomicGrid
 from grid.becke import BeckeWeights
-from grid.utils import get_cov_radii
 
 import numpy as np
 
 
 class MolGrid(Grid):
     """Molecular Grid for integration."""
 
-    def __init__(self, atomic_grids, numbers, aim_weights="becke", store=False):
-        """Initialize molgrid class.
+    def __init__(self, atomic_grids, aim_weights, store=False):
+        r"""Initialize class.
 
         Parameters
         ----------
         atomic_grids : list[AtomicGrid]
             list of atomic grid
-        radii : np.ndarray(N,)
-            Radii for each atom in the molecular grid
-        aim_weights : str or np.ndarray(K,), default to "becke"
-            Atoms in molecule weights. If str, certain function will be called
-            to compute aim_weights, if np.ndarray, it will be treated as the
-            aim_weights
+        aim_weights : BeckeWeights or np.ndarray(K,)
+            Atoms in molecule weights.
+
```

Comment:
It may convenient to still support the string API, for backward compatibility and ease of use.

Reply:
I was thinking that supporting `str` as an `aim_weights` is more like porcelain, so we can probably support it in a higher-level code.

Reply:
agreed